debug/analyzer/parse_r1cs.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import re
+
+"""
+Finds the index corresponding to `annotation`
+in the annotation set (which has the structure below)
+
+"variables_annotations":
+    [
+        {
+            "index": INDEX,
+            "annotation": ANNOTATION
+        },
+
+        ...
+
+        {
+            "index": INDEX,
+            "annotation": ANNOTATION
+        }
+    ]
+"""
+def get_index(annotation_set, annotation):
```

Comment:
Contrary to many languages, Python documentation comments usually go immediately inside the function, right at the start.  (The runtime then associates the text with `get_index.__doc__`, I think.)

Reply:
Good one, of course, thanks @dtebbs will move the comments down